### PR TITLE
Add nuTorrent

### PR DESCRIPTION
**Pull request by LeeChSien**
a BitTorrent Client

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Some good apps written with Electron.
 - [Markn](https://github.com/minodisk/markn) - Lightweight Markdown viewer.
 - [Shiba](https://github.com/rhysd/Shiba) - Live preview Markdown app with linting.
 - [Yays](https://github.com/Bahlaouane-Hamza/Yays) - YouTube music player in your menubar.
+- [Electron Torrent](https://github.com/LeeChSien/electron-torrent) - BitTorrent Client.
```
Comment (sindresorhus) [inline readme.md]: > BitTorrent client.

**Comment by sindresorhus** [comment]
Electron Torrent is IMHO not a very good or catchy name for a torrent client. It sounds more like a module you can use in your Electron app.

**Comment by LeeChSien** [comment]
Thanks for your advice.

**Comment by sindresorhus** [comment]
Probably not the best idea either, `ν` (nu) looks too much like `v`, and you're even using `nu` instead in some cases, which makes for a very inconsistent experience. I would find a name that can be consistency displayed without ambiguity.

**Comment by LeeChSien** [comment]
Finally, I decide to use "nuTorrent". I think I don't have a talent for naming (lol).

**Pull request merged**